rest/taskana-rest-spring/src/test/java/pro/taskana/task/rest/assembler/BulkOperationResultsRepresentationModelAssemblerTest.java

```diff
@@ -0,0 +1,71 @@
+package pro.taskana.task.rest.assembler;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import pro.taskana.common.api.BulkOperationResults;
+import pro.taskana.common.api.TaskanaEngine;
+import pro.taskana.common.api.exceptions.ErrorCode;
+import pro.taskana.common.api.exceptions.TaskanaException;
+import pro.taskana.common.internal.util.EnumUtil;
+import pro.taskana.rest.test.TaskanaSpringBootTest;
+import pro.taskana.task.api.TaskService;
+import pro.taskana.task.api.TaskState;
+import pro.taskana.task.api.exceptions.InvalidTaskStateException;
+import pro.taskana.task.rest.models.BulkOperationResultsRepresentationModel;
+
+@TaskanaSpringBootTest
+class BulkOperationResultsRepresentationModelAssemblerTest {
+
+  TaskanaEngine taskanaEngine;
+  TaskService taskService;
+  BulkOperationResultsRepresentationModelAssembler assembler;
+
+  @Autowired
+  BulkOperationResultsRepresentationModelAssemblerTest(
+      TaskanaEngine taskanaEngine,
+      TaskService taskService,
+      BulkOperationResultsRepresentationModelAssembler assembler) {
+    this.taskanaEngine = taskanaEngine;
+    this.taskService = taskService;
+    this.assembler = assembler;
+  }
+
+  @Test
+  void should_ReturnRepresentationModel_When_ConvertingEntityToRepresentationModel()
+      throws Exception {
+    List<String> taskIds =
```

Comment:
Why do you need taskIds here?

Reply:
You are right, I dont need them, I will delete the taskIds